jscomp/build_tests/cli_help/input.js

```diff
@@ -0,0 +1,202 @@
+// @ts-check
+
+const assert = require("assert");
+const child_process = require("child_process");
+
+const cliHelp =
+  "Usage: rescript <options> <subcommand>\n" +
+  "\n" +
+  "`rescript` is equivalent to `rescript build`\n" +
+  "\n" +
+  "Options:\n" +
+  "  -v, -version  display version number\n" +
+  "  -h, -help     display help\n" +
+  "\n" +
+  "Subcommands:\n" +
+  "  build\n" +
+  "  clean\n" +
+  "  format\n" +
+  "  convert\n" +
+  "  dump\n" +
+  "  help\n" +
+  "\n" +
+  "Run `rescript <subcommand> -h` for subcommand help. Examples:\n" +
+  "  rescript build -h\n" +
+  "  rescript format -h\n" +
+  "The default `rescript` is equivalent to `rescript build` subcommand\n" +
```

Comment:
This line is duplicated. Maybe we should remove the `rescript is equivalent to rescript build` from line 9?

Reply:
Personally I'd rather keep line 9 and remove line 26.